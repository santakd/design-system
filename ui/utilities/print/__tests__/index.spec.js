/* eslint-env jest */
import React from 'react';
import createHelpers from '../../../../jest.helpers';
const { matchesMarkup } = createHelpers(__dirname);

describe('print utilities', () => {
  it('hides rendered text only when printing', () =>
    matchesMarkup(
      <div className="slds-no-print">I am hidden when printing</div>
    ));
});
