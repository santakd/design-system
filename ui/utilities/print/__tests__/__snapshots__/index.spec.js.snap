// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`print utilities hides rendered text only when printing 1`] = `
<div
  className="slds-no-print"
>
  I am hidden when printing
</div>
`;
